spec/formatting.md

```diff
@@ -140,6 +140,7 @@ from the surrounding text.
 To allow for _function handlers_ to ensure that certain _option_ values are set by _literals_,
 the _resolved value_ of each _option_ value MUST include information about 
 whether the _option_ value is a _literal_ or a _variable_.
+Note that this information is irrelevant for a _resolved value_ that is not used as the value of an _option_.
```

Comment:
Hmm. Is it perhaps not clear that "_option_ value" is a syntax feature? This phrase seems to imply that a _resolved value_ could be an _option_ value, and that doesn't really make sense.

We probably ought to explicitly define _option value_ as a term in the Syntax section.

Reply:
> We probably ought to explicitly define option value as a term in the Syntax section.

Absolutely!

Reply:
Let's do that in a separate PR.